apps/condo/domains/organization/tasks/syncSbbolBankAccounts.js

```diff
@@ -32,9 +32,11 @@ async function syncSbbolBankAccounts () {
     const integration = await BankIntegration.getOne(context, { id: BANK_INTEGRATION_IDS.SBBOL })
     if (!integration) throw new Error(`Cannot find SBBOL integration by id=" ${BANK_INTEGRATION_IDS.SBBOL}"`)
 
-    for (const identity of usersWithSBBOLExternalIdentity) {
+    for (let identity of usersWithSBBOLExternalIdentity) {
         const [employee] = await OrganizationEmployee.getAll(context, {
-            user: { id: identity.user.id },
+            user: {
+                id: identity.user.id,
+            },
```

Comment:
It it possible to fetch users with one request, using `id_in` query field?

Reply:
yes